salt/kubelet/kubelet.jinja

```diff
@@ -15,7 +11,7 @@ KUBELET_PORT="--port=10250"
 KUBELET_HOSTNAME="--hostname-override={{ grains['id'] }}.{{ pillar['internal_infra_domain'] }}"
 
 # location of the api-server
-KUBELET_API_SERVER="--api-servers={{ api_server_url }}"
+KUBELET_API_SERVER="--api-servers=https://api.{{ pillar['internal_infra_domain'] }}:6444"
```

Comment:
same as `salt/kubeconfig/kubeconfig.jinja` - we should leave this file as is, and update pillars.